src/components/Link/link.module.scss

```diff
@@ -1,15 +1,95 @@
 .link-style {
   cursor: pointer;
+  display: inline-block;
   font-family: $octuple-font-family;
   font-size: $text-font-size-2;
+  text-decoration: underline;
```

Comment:
based on a11y related chat with Yair, seems like we want to give anchors a slightly heavier font weight, and only underline on hover/focus

Reply:
Will add semibold style, I used an css approach to underline, so the second bit of feedback is what's implemented. However, the default is preserved as underlined in those cases where engineers just need a plain-old link.

Reply:
discussed with @ypike-eightfold - updated to semibold.